internal/database/clickhouse/clickhouse_test.go

```diff
@@ -0,0 +1,24 @@
+package clickhouse
+
+import (
+	_ "github.com/ClickHouse/clickhouse-go"
+
+	"database/sql"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+//docker run -d --rm --name clickhouse-server -p 8123:8123 --ulimit nofile=262144:262144 yandex/clickhouse-server
```

Comment:
Can we use here docker mock for tests?
https://github.com/testcontainers/testcontainers-go